app/mm-redux/selectors/entities/preferences.test.js

```diff
@@ -181,6 +210,7 @@ describe('Selectors.Preferences', () => {
                         config: {
                             DefaultTheme: 'default',
                         },
+                        serverVersion: 'dev',
```

Comment:
what is the reason behind this ? if you need a minimum server version, just set it here instead and add a comment as to why